### DISCUSSION TITLE
Binding port different port in the container?

### DISCUSSION THREAD

**Discussion by cchance27** [jellyfin/jellyfin-vue#2222] [Questions]
When running in a rootless podman, the container for vue seems to bind to port 80 which results in a 
2024/02/04 16:25:52 [emerg] 1#1: bind() to 0.0.0.0:80 failed (13: Permission denied)
nginx: [emerg] bind() to 0.0.0.0:80 failed (13: Permission denied)

any chance that this is exposed as a env variable somewhere that we can have it bind to a higher port in the container, and we can handle the redirect outside?

**Comment by GETandSELECT** [comment]
Bump. Got the same Issue with podman, maybe open an Issue?

**Reply by ferferga** [reply]
Withe the latest unstable image too?

**Comment by ferferga** [comment]
No further input was given and https://github.com/jellyfin/jellyfin-vue/pull/2247 fixes a similar issue, so I'm closing this.
